src/types/Particle.ts

```diff
@@ -76,9 +68,5 @@ export const PARTICLE_TYPES = [
 	{
 		id: 24,
 		name: 'Helium-4'
-	},
```

Comment:
According to this comment:
https://github.com/yaptide/ui/issues/1357#issuecomment-1837227357

the heavy ions work when Z and A is provided. I'd keep heavy ions.